data/brands/shop/window_blind.json

```
@@ -4,6 +4,17 @@
     "exclude": {"generic": ["^window blind$"]}
   },
   "items": [
+    {
+      "displayName": "Blinds to Go",
+      "id": "blindstogo-a6607a",
+      "locationSet": {"include": ["ca"]},
```

```suggestion
      "locationSet": {"include": ["ca-on.geojson", "us-ct.geojson", "us-de.geojson", "us-ma.geojson", "us-md.geojson", "us-nj.geojson", "us-ny.geojson", "us-pa.geojson", "us-va.geojson"]},
```
per https://www.blindstogo.com/en/stores